### ISSUE TITLE
Lots of warnings building 1.4.x on macs

### ISSUE THREAD

**Issue by erco77** [fltk/fltk#590]
Building 1.4.x with cmake/make on both macos 11.5.2 and 10.10.5, seeing a lot of warnings about "friend Fl_System_Driver", e.g.

```
/var/tmp/fltk-now/FL/Fl.H:138:10: warning: unelaborated friend declaration is a C++11 extension; specify 'class' to befriend 'Fl_System_Driver' [-Wc++11-extensions]
  friend Fl_System_Driver;
         ^
         class
```
Since so many files include Fl.H, this warning shows up *a lot* (several hundred times).

I assume this is easy to replicate since I get it on mac os releases that are so far apart, but I also see these in the gitlab logs for recent commits.

**Comment by ManoloFLTK** [comment]
That should be fixed with commit 19c1ab5

**Comment by erco77** [comment]
I looked thru the gitlab build for Manolo's fix and the warnings are gone, so looks good.
